core/client/app/utils/link-component.js

```diff
@@ -6,8 +6,8 @@ LinkComponent.reopen({
     active: computed('attrs.params', '_routing.currentState', function () {
```

Comment:
does the 'attrs.params' need to be here because we're not using attrs anymore? or does it need to be something else?